tests/airport.spec.js

```diff
@@ -153,7 +153,7 @@ expected = 5;
 
 // Act
 airport.setCapacity(5);
-for (let i = 0; i <= airport.maximumCapacity; i++) {
```

Comment:
This was foolish as all this would've done is run the test to the maximum capacity set and not actually try and push a plane onto the airport array beyond the set capacity. I realised this and amended the test.